tir/authorizationClient.go

```diff
@@ -35,13 +37,34 @@ type NoAuthHttpClient struct {
 	httpClient HttpClient
 }
 
+func (ac AuthorizingHttpClient) FillMetadataCache(context.Context) {
+	tirEndpointsInterface, hit := common.GlobalCache.TirEndpoints.Get("tirEndpoints")
+	if !hit {
+		logging.Log().Info("issuers list not found in cache")
```

Comment:
Wouldn't this be an unexpected error if we have a tir endpoint referenced but don't have it here?

Reply:
Here it could happen, since tir endpoints cache is updated from another scheduling task. But it'll be unexpected, when the metadata is being read from cache and it's missing. In that case, it'll throw an error